### DISCUSSION TITLE
Semver tagged versions on container images for River UI

### DISCUSSION THREAD

**Discussion by hyrefeldt-ourstudio** [riverqueue/riverui#185] [Q&A] [answered]
Are there any plans to tag the container images for the River UI in `vX.X.X` format? Right now I need to use the long sha hash to fix the image version that I use for the UI. Would be nice with a semver version tag instead.

**Comment by bgentry** [comment, marked as answer]
They’re already here, but without the v prefix. As far as I could tell this was conventional in the Docker world. GitHub ui doesn’t make them easy to find alongside all the SHA256 tags though 😕 https://github.com/riverqueue/riverui/pkgs/container/riverui/269067762?tag=0.5.3

**Reply by hyrefeldt-ourstudio** [reply]
Aha great! Thanks for the help. I will look into it.

**Reply by hyrefeldt-ourstudio** [reply]
Works fine!
